api/enterprise-api-ref/users-rbac.markdown

```diff
@@ -367,3 +367,28 @@ API call allowed only for administrator.
 
 Remove role definition.
 API call allowed only for administrator.
+
+## Unlock user
+
+If a system enforces using two-factor authentication (2FA),
+users must configure it after their first login within 48 hours.
+This endpoint unlocks users who have been locked out due to this reason
+and grants an additional 48 hours to complete the 2FA setup
```

Comment:
Who can perform this unlock?

Can unlock be performed an unlimited number of times?